docs/webmcp-sdk/page-agent-tool.md

```
@@ -147,8 +172,13 @@ import { resolveA11yInfo } from '@opentiny/next-sdk'
 
 resolveA11yInfo(document.querySelector('.tab-item')!, a11yConfig)
 // { role: 'tab', tokens: ['selected'] }
+
+resolveA11yInfo(document.querySelector('ti-app-layout-left')!, a11yConfig)
+// { role: 'navigation', tokens: [...], name: '侧边导航' }
 ```
 
+> 命中带 `name` 的 role 规则时，`resolveA11yInfo` 会多返回可选字段 `name`（与 YAML 中的声明可访问名一致）。
+
```

markdownlint-cli2 (0.23.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)